api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -250,13 +250,22 @@ message OktaAssignmentMetadata {
   string EndingStatus = 4 [(gogoproto.jsontag) = "ending_status,omitempty"];
 }
 
+// AccessListMetadata contains common metadata for access list resource events.
+message AccessListMetadata {
+  // AccessListTitle is a plain text short description of the access list.
+  string AccessListTitle = 1 [(gogoproto.jsontag) = "access_list_title,omitempty"];
+}
+
 // AccessListMemberMetadata contains common metadata for access list member resource events.
 message AccessListMemberMetadata {
   // AccessListName is the name of the access list the members are being added to or removed from.
   string AccessListName = 1 [(gogoproto.jsontag) = "access_list_name,omitempty"];
 
   // Members are all members affected by the access list membership change.
   repeated AccessListMember Members = 2 [(gogoproto.jsontag) = "members,omitempty"];
+
+  // AccessListTitle is a plain text short description of the access list the members are being added to or removed from.
+  string AccessListTitle = 3 [(gogoproto.jsontag) = "access_list_title,omitempty"];
```

Comment:
That's a bit strange to include AccessListTitle inline in all these events. You already have AccessListMetadata defined above - can we instead just include that metadata object in all relevant events? Then it will be easier to extend do.

Reply:
I looked at it differently. I thought all `AccessList{verb}` events would have `AccessListMetadata AccessList` field and all `AccessListMember{verb}` events `AccessListMemberMetadata AccessListMember` field.

I can add `AccessListMetadata AccessList` to `AccessListMember{verb}` if you prefer. Please let me know what you think.